front/src/modules/simulationResult/consts.ts

```diff
@@ -51,3 +51,8 @@ export type PositionScaleDomain = {
   current: number[];
   source?: 'SpeedSpaceChart' | 'SpaceCurvesSlopes';
 };
+
+export type TimeScaleDomain = {
+  range?: [Date, Date];
+  source?: 'SpaceTimeChart' | 'Timeline';
+};
```

Comment:
According to this morning's meeting, should this be put in a types.ts file ?

Reply:
You're right ! Done